bin/check-load.rb

```diff
@@ -28,11 +28,15 @@
 class LoadAverage
   def initialize(per_core = true)
     @cores = per_core ? cpu_count : 1
-    @avg = File.read('/proc/loadavg').split.take(3).map { |a| (a.to_f / @cores).round(2) } rescue nil # rubocop:disable RescueModifier
+    @avg = `uptime`.delete(',').split(' ')[-3..-1].map { |a| (a.to_f / @cores).round(2) } rescue nil # rubocop:disable RescueModifier
```

Comment:
is there an alternate file we can use? I tend to not be a fan of shelling out. If not I would prefer to test file existence and fallback to shelling out.